extensions/ql-vscode/src/remote-queries/run-remote-query.ts

```diff
@@ -43,7 +43,7 @@ interface QueriesResponse {
  * - `owner` is made up of alphanumeric characters or single hyphens, starting and ending in an alphanumeric character
  * - `repo` is made up of alphanumeric characters, hyphens, or underscores
  */
-const REPO_REGEX = /^(?:[a-zA-Z0-9]+-)*[a-zA-Z0-9]+\/[a-zA-Z0-9-_]+$/;
+export const REPO_REGEX = /^(?:[a-zA-Z0-9]+-)*[a-zA-Z0-9]+\/[a-zA-Z0-9-_]+$/;
```

Comment:
Probably good to move this to a new file. I don't think `databaseFetcher` should be depending on `run-remote-query`. Maybe `helpers` or `helpers-pure`.

Reply:
Done! ✔️